knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -229,7 +229,7 @@ public Response addDocumentTemplate(String label, MultiPartBody input) {
 
 	}
 
-	public Response deleteCurrentTemplate(String documentLabel) {
+	protected Response deleteCurrentTemplate(String documentLabel) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java.